WalletWasabi/WabiSabi/Client/ArenaClient.cs

```diff
@@ -64,6 +65,34 @@ public ArenaClient(WabiSabiClient amountCredentialClient, WabiSabiClient weightC
 			return inputRegistrationResponse.AliceId;
 		}
 
+		public async Task RemoveInputAsync(Guid roundId, Guid aliceId)
```

Comment:
No tests for this one.

Reply:
It is a pass-through method, the code is already covered. Anyway, i will add one..

Reply:
Done.